tests/suites/machine/machine.sh

```diff
@@ -0,0 +1,32 @@
+test_log_permissions() {
+    # Echo out to ensure nice output to the test suite.
+    echo
+
+    # The following ensures that a bootstrap juju exists
+    file="${TEST_DIR}/test_log_permissions.txt"
+    ensure "correct-log" "${file}"
+
+    juju deploy postgresql
+
+    wait_for "started" '.machines."0"."juju-status".current'
+
+    check_contains "$(juju ssh 0 -- stat -c '%G' /var/log/juju/unit-postgresql-0.log)" adm
```

Comment:
Probably want to check machine log file too.

Reply:
:+1: